static/scss/dashboard.scss

```diff
@@ -251,3 +251,25 @@ c.validation-wrapper {
 .education-icons {
   text-align: right;
 }
+
+.terms-of-service-main {
+  width: 75%;
+  margin-left: auto;
+  margin-right: auto;
+  div.terms-card-header {
+    padding: 25px 0 25px 0;
```

Comment:
If you're going to have the same horizontal and vertical padding you should format it like this: `padding: 25px 0;`